src/components/TaskList.vue

```diff
@@ -54,11 +61,26 @@ export default {
     border: none;
   }
 
-    &--completed {
+  &--completed {
     button {
       background: rgba(145, 255, 145, 0.5);
       color: white;
     }
   }
 }
-</style>
\ No newline at end of file
+
+.task-list-item__title {
+  text-align: start;
+}
+
+.task-list-item__wiki {
+  text-decoration: none;
+  color: rgba(145, 255, 145, 0.5);
+}
+
+.task-list-item__wiki:hover {
+  text-decoration: underline;
+}
+.c {
+}
```

Comment:
This class is being applied to task list item title, however it's empty so can we remove this?